src/static/js/repl.js

```diff
@@ -0,0 +1,108 @@
+const initialCode = `// import
```

Comment:
this file can possibly be deleted now. because our REPL is now implemented with CodeSandbox. please double check, I could be wrong.

Reply:
Yes, you are right. I deleted it.